lib/adyen/api.rb

```diff
@@ -1,4 +1,3 @@
-require 'adyen'
```

Comment:
Makes sense. I moved the code in `lib/adyen.rb` to `lib/adyen/base.rb`, and updated some more dependencies to allow independent loading.